src/gui/owncloudgui.cpp

```diff
@@ -70,17 +70,15 @@ void setUpInitialSyncFolder(AccountStatePtr accountStatePtr, const QString &loca
     if (accountStatePtr->account()->capabilities().spacesSupport().enabled) {
         auto *drive = new OCC::GraphApi::Drives(accountStatePtr->account());
 
-        QObject::connect(drive, &OCC::GraphApi::Drives::finishedSignal, [accountStatePtr, localFolder, drive, addFolder, finalize] {
+        QObject::connect(drive, &OCC::GraphApi::Drives::finishedSignal, [accountStatePtr, drive, addFolder, finalize] {
             if (drive->parseError().error == QJsonParseError::NoError) {
                 const auto &drives = drive->drives();
                 if (!drives.isEmpty()) {
-                    const QDir localDir(localFolder);
-                    localDir.mkdir(".");
-                    FileSystem::setFolderMinimumPermissions(localFolder);
-                    Utility::setupFavLink(localFolder);
+                    const QDir localDir(accountStatePtr->account()->defaultSyncRoot());
```

Comment:
As far as I can see, there is no guarantee that the directory has not been removed in the meantime. Do we need to handle this case?

Reply:
We reach that code directly after the account was setup but well I won't hurt.